include/avt_vimba_camera/mono_camera.h

```diff
@@ -43,16 +43,20 @@
 #include <camera_info_manager/camera_info_manager.h>
 #include <image_transport/image_transport.h>
 #include <dynamic_reconfigure/server.h>
-
+#include <cav_msgs/SystemAlert.h>
+#include <cav_msgs/DriverStatus.h>
 #include <string>
 
 namespace avt_vimba_camera {
 class MonoCamera {
  public:
   MonoCamera(ros::NodeHandle& nh, ros::NodeHandle& nhp);
   ~MonoCamera(void);
-
+ void publish_status();
+ void alertCallback(const cav_msgs::SystemAlertConstPtr &msg);
```

Comment:
Please reevaluate the type of access modifiers for function ```alertCallback``` and variable ```flag```. Also it would be nice if you can give a more meaningful name to the ```flag``` variable.